00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
```

Comment:
Good catch! We want "categories" rather than category on all activities that we post.

categories: [self-paced-tutorial]

I need to update the template in this repo and then we can test this change to ensure it works properly.
I have pasted the YAML that I see in L00. It looks right to me with the exception of category should be categories: [self-paced-tutorial] . What workshop tag are you referring to? The maintag is OK. This is the hack I was telling you about to make the content populate on the bottom of the page as a workshop. We can fix this when we've settled on the overal structure that we discussed this morning.


category: self-paced-tutorial
mainTag: vector-data-series
tags: [vector-data, R, spatial-data-gis]
workshopSeries: [vector-data-series]